### PR TITLE
Improve contributing

### PR DESCRIPTION
**Pull request by lamiazar**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,55 +1,42 @@
...
-cd socket
-./scripts/install.sh
+cd socket 
+./bin/install.sh
 ```
 
-Windows:
-```powershell
+### Windows
+```bash
```
[the remark this answers is not in the document]
Comment (lamiazar) [reply CONTRIBUTING.md]: corrected

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,55 +1,43 @@
 ## Contribution Guide Overview
-This guide provides steps on how to make contributions and become a part of the ongoing development of `Socket Runtime`.  
+This guide provides steps on how to make contributions, report bugs, and become a part of the ongoing development of `Socket Runtime`.  
+If you are just started learning about Socket Runtime, you can start from [here](https://socketsupply.co/guides/).
```
Comment (bcomnes) [inline CONTRIBUTING.md:3]:
```suggestion
If you are new to the Socket Runtime project, the [guide](https://socketsupply.co/guides/) is a good place start.  
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,55 +1,43 @@
 ## Contribution Guide Overview
-This guide provides steps on how to make contributions and become a part of the ongoing development of `Socket Runtime`.  
+This guide provides steps on how to make contributions, report bugs, and become a part of the ongoing development of `Socket Runtime`.  
+If you are just started learning about Socket Runtime, you can start from [here](https://socketsupply.co/guides/).  
 
-### Requirements
+## Setup and build the framework
```
Comment (bcomnes) [inline CONTRIBUTING.md:5]:
```suggestion
## Requirements
```
Changing the header order is fine, but I think "Requirements" is more idiomatic.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,55 +1,43 @@
...
 
-To contribute to the project you need to clone the repository and install the dependencies. You can do this by running the following commands:
+To contribute to the project you need to clone the main repository first and install the dependencies.
```
Comment (bcomnes) [inline CONTRIBUTING.md:7]:
```suggestion
To contribute to the project you need to clone the repository and install the dependencies. 
```
"main" and "first" read redundant.

**Review by bcomnes — COMMENTED** [review]
Left a few suggestions.

**Comment by lamiazar** [comment]
I will apply the changes, thank you

**Review by bcomnes — APPROVED** [review]

**Review by jwerle — APPROVED** [review]

**Pull request merged**